### DISCUSSION TITLE
🚀 Framework NIS2 disponible maintenant ! / NIS2 Framework Now Available!

### DISCUSSION THREAD

**Discussion by dbarzin** [dbarzin/deming#139] [Announcements]
# 🚀 Framework NIS2 maintenant disponible !!!

Nous sommes heureux d'annoncer l'ajout du framework **NIS2** dans Deming ! 🎉

## Qu'est-ce que le NIS2 ?
Le **NIS2** est une directive européenne sur la sécurité des réseaux et des systèmes d'information. Ce framework vise à renforcer la résilience des systèmes critiques à travers l'Union européenne, en mettant l'accent sur la cybersécurité des infrastructures essentielles. Il apporte des lignes directrices sur la gestion des risques, la sécurité des systèmes, la gestion des incidents et la conformité réglementaire.

### Fonctionnalités principales :
- Politiques d'analyse des risques intégrées
- Gestion des incidents basée sur le NIS2
- Conformité renforcée avec les exigences européennes en matière de sécurité
- Plan d'actions correctives basé sur les mesures de sécurité

## Comment l'utiliser ?
Pour activer le framework NIS2 allez dans __Configuration__ -> __Import__, choisissez **NIS2.fr** et cliquez sur __Importer__ pour ajouter le framework ! 

---

# 🚀 NIS2 Framework Now Available!!!

We are excited to announce the addition of the **NIS2** framework in Deming! 🎉

## What is NIS2?
The **NIS2** is a European directive on the security of network and information systems. This framework aims to strengthen the resilience of critical systems across the European Union, with a strong focus on the cybersecurity of essential infrastructures. It provides guidelines on risk management, system security, incident management, and regulatory compliance.

### Key Features:
- Integrated risk analysis policies
- Incident handling based on NIS2
- Enhanced compliance with European security requirements
- Action plans based on security measures

## How to use it?
To activate the NIS2 framework, go to __Configuration__ -> __Import__, choose **NIS2.en** and click __Import__ to add the framework!

**Comment by Canela352** [comment]
Hello Mr Barzin

I hope you are doing well.
I did try to instal NIS2 in Deming but I dont find it among the available options. 
Could you please let me know where could I get it? 

best regards
![image](https://github.com/user-attachments/assets/e36f4b99-5c5f-45a2-95b1-1f22c0fcf67d)

**Comment by dbarzin** [comment]
You have to update the application https://github.com/dbarzin/deming/blob/main/INSTALL.md#update
or you may download the NIS2 framework from here https://github.com/dbarzin/deming/tree/main/storage/app/repository

**Reply by Canela352** [reply]
Thank you! :)
